resources/assets/js/components/TopBar.vue

```diff
@@ -31,14 +31,13 @@
 </template>
 
 <script>
-import { mapState, mapMutations } from 'vuex';
-import { Loading } from 'element-ui';
+import { mapState } from 'vuex';
 import Icon from 'components/Icon';
 import { undoStackInstance } from 'plugins/undo-redo';
 import { onKeyDown, onKeyUp } from 'plugins/key-commands';
 import Toolbar from 'components/Sidebar/Toolbar';
 
-/* global window */
+/* global window, document */
```

Comment:
Adding references to "globals" for eslint. This is also useful if we want to server-side rendering (SSR) as it gives us a quick overview of browser specific APIs.